src/github/push.ts

```diff
@@ -23,6 +23,11 @@ export const pushWebhookHandler = async (context: WebhookContext, jiraClient): P
 		installation: context.payload?.installation
 	};
 
+	context.log = context.log.child({
+		jiraHostName: jiraClient.baseURL,
```

Comment:
why `jiraHostName` where we have `jiraHost` everywhere else?

Reply:
(⌒_⌒;) - have updated all jirahostnames